server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -114,6 +116,16 @@ fn generate_response_requisition(
         comment: None,
     };
 
+    system_log_entry(
+        connection,
+        match result.status.clone() {
+            RequisitionRowStatus::New => LogType::RequisitionCreated,
+            _ => LogType::RequisitionCreated,
+        },
```

Comment:
Should the default arm be doing something else?
If not - remove the match?

Reply:
All the patterns need to be handled, so can't remove the wildcard... but can change it to a different LogType if you want?

Reply:
I think Mark is asking, why is there a match ?

Reply:
yep. the `LogType::RequisitionCreated` is returned if it is `RequisitionRowStatus::New` 
If it isn't `LogType::RequisitionCreated` then the return is also `RequisitionRowStatus::New` 
there is no need for the match - you can just set it to `LogType::RequisitionCreated` 😉 

// perhaps a comment to explain why the type is always `RequisitionCreated` though?

I was wondering though, if the wildcard was supposed to be returning a different value?

Reply:
🤦